tests/test_functions.py

```diff
@@ -114,6 +114,21 @@ def test_extract_hashbits_info():
             print >>sys.stderr, '...failed to remove {fn}'.format(fn)
 
 
+def check_file_status_kfile():
+    try:
```

Comment:
generate a guaranteed-not-to-exist filename with utils.get_temp_filename('thisfiledoesnotexist')